test/test_fx.py

```diff
@@ -1597,6 +1598,62 @@ def test_no_mutation(self):
         with self.assertRaisesRegex(NotImplementedError, "new_args"):
             x[0] = 4
 
+    def test_custom_traceback_raised_when_exception_source_is_graphmodule(self):
+        class M(torch.nn.Module):
+            def __init__(self):
+                super(M, self).__init__()
+                self.W = torch.nn.Parameter(torch.randn(5))
+
+            def forward(self, x):
+                return torch.dot(self.W, x)
+
+        traced = torch.fx.symbolic_trace(M())
+
+        out_nodes = [n for n in traced.graph.nodes if n.op == "output"]
+        out = out_nodes[-1]
+        relu_out = traced.graph.call_method(method_name='relu', args=(out.args[0],))
+        out.replace_all_uses_with(relu_out)
```

Comment:
Hate to keep commenting on this test case, but two things:
1) I think you want to insert the `call_method` call before the output node
2) Similarly to args above, I think you should do `out.args[0].replace_all_uses_with(relu_out)`